Dockerfile

```diff
@@ -37,5 +37,8 @@ RUN curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl \
 RUN chmod +x ./kubectl
 RUN mv ./kubectl /usr/local/bin
 
+# Addition of 'll' alias
+RUN echo "alias ll='ls -al'" >> ~/.bash_profile
```

Comment:
Don't do this here. Run the alias command every time you exec into the container, `kubectl cp` your bashrc manually before running your commands, or write a wrapper script locally that passes stdin to the container using `--stdin`.

If you must put personal preferences for a specific developer into the global codebase, mount `~/.bash_profile` [via a configmap](https://kubernetes.io/docs/tasks/configure-pod-container/configure-pod-configmap/#populate-a-volume-with-data-stored-in-a-configmap) instead.

Reply:
Removed the alias.